src/skmultiflow/anomaly_detection/six_sigma.py

```diff
@@ -0,0 +1,81 @@
+# using six sigma to determine the standard deviation of data
+
+#import statistics as stat
+import numpy as np
+import matplotlib.pyplot as plt
+
+class SixSigma ():
+
+    def __init__(self,time_window=2):
+
+        self.time_window = time_window
+        self.buffer = []
+
+
+    def fit (self,X,y):
+        self.buffer.append(X)
+        self.buffer = self.buffer[-self.time_window:]
+        #print(buffer)
+
+    def predict (self,X,element):
+
+        if len(self.buffer) < self.time_window:
+            print(f'element {element} y_pred None')
+            return None
+        else:
+            j = 0
+            while j < len(X):
+                i = 0
+                feature_array =[]
+                while i < len(self.buffer):
+                    feature_array.append(self.buffer[i][j])
+                    i += 1
+                average = np.mean(feature_array)
+                sigma = np.std(feature_array)
+                print(f'element {element}')
+                print(f'array {feature_array}')
+                print(f'mean {average}')
+                print(f' sigma {sigma}')
+                if abs(X[j]) > abs(average) + 3*abs(sigma):
```

Comment:
prediction = int(abs(X[j]) > abs(average) + 3*abs(sigma))
j+=1

Pa se izogneš if else in vse ostalo